doctr/models/detection/differentiable_binarization.py

```diff
@@ -403,24 +404,30 @@ def draw_thresh_map(
             canvas[ymin_valid:ymax_valid + 1, xmin_valid:xmax_valid + 1]
         )
 
-    def compute_target(
+    def compute_loss(
         self,
-        out_shape: Tuple[int, int, int, int],
+        model_output: Dict[str, tf.Tensor],
         batch_polys: List[List[List[List[float]]]],
         to_masks: List[List[bool]]
```

Comment:
I highly suggest renaming this to batch_flags (in computer vision, masks usually refers to semantic or binary spatial masks)